sdk/logs/build.gradle.kts

```diff
@@ -28,6 +28,7 @@ testing {
       dependencies {
         implementation(project(":sdk:testing"))
         implementation(project(":api:incubator"))
+        implementation("io.opentelemetry.semconv:opentelemetry-semconv-incubating")
```

Comment:
I didn't notice which incubating semconv attribute is needed in the tests?

Reply:
None of the incubating ones, but the stable test references `ExceptionAttributes`. Cheating by depending on `opentelemetry-semconv-incubating` instead of `opentelemetry-semconv` because that's the artifact we manage in dependencyManagement: https://github.com/open-telemetry/opentelemetry-java/blob/main/dependencyManagement/build.gradle.kts#L87